zipline/assets/roll_finder.py

```diff
@@ -90,6 +90,7 @@ def get_rolls(self, root_symbol, start, end, offset):
                                         tc.minute_to_session_label(end))
         freq = sessions.freq
         if first == front:
+            # Compute at least one roll date no matter what.
```

Comment:
Thanks for this comment, mind adding a little more detail on how this follows from the code? I still have to think about it for a while even though I've had a lot of exposure to this